Update Presence Information (#8555)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/teams-and-skypeforbusiness-coexistence-and-interoperability.md

For example, Assume User A is in Islands mode. If User A is active in Teams but is not signed in to Skype for Business, other users would see User A as active from their Teams client, but in their Skype for Business client they would see User A as offline. This is by design, since User A cannot be reached if they are not running the client. 

> Presence is not supported for Sfb users who have SfbOnly or similar mode while logged into a Teams client.

Reviewer: ```suggestion
Presence is not supported for Skype for Business users who are in SfbOnly or similar mode while logged into a Teams client.
```
